### PR TITLE
docs: update python document

### PR DESCRIPTION
**Pull request by sunpe**
docs: add docs for assignment and seek
docs: upgrade docs from python subscription
docs: support stmt via ws
docs: optimizate python document base on java document

### CODE REVIEW

**Review by AdamEECS — APPROVED** [review]
LGTM

File: docs/zh/08-connector/30-python.mdx [hunk 1]
```diff
@@ -456,27 +530,169 @@ TaosCursor 类使用原生连接进行写入、查询操作。在客户端多线
...
+
+```python
+from taos.tmq import Consumer
+
+consumer = Consumer({"group.id": "local", "td.connect.ip": "127.0.0.1"})
+```
+
+#### 订阅 topics
+
+Comsumer API 的 `subscribe` 方法用于订阅 topics，consumer 支持同时订阅多个 topic。
```
Comment (sangshuduo) [inline docs/zh/08-connector/30-python.mdx:552]: Consumer

File: docs/zh/08-connector/30-python.mdx [hunk 2]
```diff
@@ -456,27 +530,169 @@ TaosCursor 类使用原生连接进行写入、查询操作。在客户端多线
...
+
+```python
+import taosws
+
+consumer = taosws.(conf={"group.id": "local", "td.connect.websocket.scheme": "ws"})
+```
+
+#### 订阅 topics
+
+Comsumer API 的 `subscribe` 方法用于订阅 topics，consumer 支持同时订阅多个 topic。
```
Comment (sangshuduo) [inline docs/zh/08-connector/30-python.mdx:634]: Consumer

**Review by sangshuduo — CHANGES_REQUESTED** [review]
typos

**Review by gccgdb1234 — APPROVED** [review]

**Pull request merged**